curriculum/challenges/english/25-front-end-development/quiz-javascript-arrays/66edcccbba6dacdb65a59067.md

```diff
@@ -17,439 +17,529 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+What is the correct way to declare a JavaScript array?
 
 #### --distractors--
 
-Placeholder distractor 1
+`let arr = array(1, 2, 3);`
 
 ---
 
-Placeholder distractor 2
+`let arr = "1, 2, 3"; `
 
 ---
 
-Placeholder distractor 3
+`let arr = {1, 2, 3};`
 
 #### --answer--
 
-Placeholder answer
+`let arr = [1, 2, 3];`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the key difference between primitive and non-primitive data types?
 
 #### --distractors--
 
-Placeholder distractor 1
+Primitives can store multiple values.
 
 ---
 
-Placeholder distractor 2
+Non-primitives cannot hold references to other objects.
 
 ---
 
-Placeholder distractor 3
+Non-primitives store values directly.
 
 #### --answer--
 
-Placeholder answer
+Primitives hold direct values, non-primitives hold references.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What will be the result of the following array method `indexOf()`?
+
+
+```javascript
+const numbers = [10, 20, 30, 40];
+const index = numbers.indexOf(20);
+console.log(index);
+```
```

Comment:
````suggestion
What will be the result of the following array method `indexOf()`?

```js
const numbers = [10, 20, 30, 40];
const index = numbers.indexOf(20);
console.log(index);
```
````

I think 'javascript' is also accepted by the linter, but we usually use 'js'